x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/querydsl/query/SpatialRelatesQuery.java

```diff
@@ -220,37 +209,15 @@ org.apache.lucene.search.Query buildShapeQuery(SearchExecutionContext context, M
         private static org.apache.lucene.search.Query pointShapeQuery(
```

Comment:
The comment above this method is no longer valid, since this code no longer is based on that code, but now calls directly the newer version of that code. I had extended this method handle disjoint and contains, but your newer code also handles those cases, and more, so presumably entirely subsumes this method.

Reply:
removed